event.go

```diff
@@ -357,6 +358,7 @@ func (e *Event) populateFieldsFromJSON(eventJSON []byte) error {
 		if eventJSON, err = sjson.DeleteBytes(eventJSON, "event_id"); err != nil {
 			return err
 		}
+		e.eventJSON = eventJSON
```

Comment:
Why are we setting this each time for each format and not doing it before the `switch`?

Reply:
Oh because this version doesn't have an event ID.